### ISSUE TITLE
Submit a funded issue (or send a tip) and provide feedback on the UX.

### ISSUE THREAD

**Issue by owocki** [gitcoinco/web#37]
Requirements:
* Submit a funded issue, or a tip to the platform.
* Write _thoughtful_ feedback on what you thought of the process, where you got confused.  What should be changed about the process?
* Ill payout this bounty to no more than 4 people.

**Comment by gitcoinbot** [comment]
__This issue now has a funding of 0.09 ETH (26.47 USDT) attached to it.__  To view or claim this funding, [click here](https://gitcoin.co/funding/details?url=https://github.com/gitcoinco/web/issues/37).

**Comment by chunkzer** [comment]
I've added a funded issue to a gem I use at work: https://github.com/adore/daylight/issues/17
Here are my thoughts on the process: 

In the "Fund Issue" form things are all pretty great except I'd like to see an ETH to USD conversion  in the amount row. I'll get to see it anyways when metamask fires up but I think it would help anyway. 

I don't see much of a reason for the advanced options dropdown, yet. The advanced features are very few and not that mind-bending to warrant them being hidden right off the bat. 

Especially since: 

1. Issue expiration should be very important for the funder to understand clearly. Who'd want to escrow ETH for over a year!? (default?) Could lead to some bad experiences.

2. The meta data in issue details is like the SEO of Gitcoin. If you want to get your issue solved you'd really need to optimize that.

I've funded my issue for a week. I'm not a repo owner and I wonder if submitting this type of issue still has any value. I do have a fork of daylight which would probably had been a much better place but oh well. 

I'll update with anything else I think about as the bounty runs through its life cycle.

**Comment by owocki** [comment]
> In the "Fund Issue" form things are all pretty great except I'd like to see an ETH to USD conversion in the amount row. I'll get to see it anyways when metamask fires up but I think it would help anyway.

This is a great idea! 

> Issue expiration should be very important for the funder to understand clearly. Who'd want to escrow ETH for over a year!? (default?) Could lead to some bad experiences.

What do you think the default should be?

**Comment by gitcoinbot** [comment]
⚡️ A tip worth 0.09  ETH has been granted to @chunkzer for this issue from @owocki. ⚡️ 

Nice work @chunkzer, check your email for further instructions. | <a href='https://gitcoin.co/tip'>Send a Tip</a>

**Comment by TimVanMourik** [comment]
I've been wanting to get started with GitCoin, so this is the full account from a complete GitCoin beginner:

-From the newsletter, I’m redirected to the GitCoin website. However, it gives a loading screen.
-I open the same page in Chrome and it clearly says that I need to install MetaMask. A clear error message, where Safari only gives a loading screen.
@suggestion: Make the 'install MetaMask' error work in Safari.
-I try to move on with Safari, my preferred browser, but MetaMask is only available on Chrome.
@suggestion: Make GitCoin work with Safari, or intuitively show how if it's already possible
-I install the MetaMask extension to Chrome, but don't know what it does
@suggestion: In the MetaMask error, explain in one sentence why it's needed
-I want to fund an issue so click on "Fund Issue". It says I need Ether. The form explains where I can get Ether (already got some in CoinBase) but not in what account I need it. Via installing the GitCoin extension, I stumbled on the address, to which I transferred ETH. Now it works automatically!
@suggestion: Explain in what account the ETH needs to be at the moment of submitting a new issue
-I click 'Fund Issue' and it's waiting... I don't quite know what it's waiting for until I realise that I need to confirm the transaction. 
@suggestion: A message/explanation/waiting screen might be nice. 

In general, some clarification along the road would be appreciated. And it's still not quite clear to me what people have to do to claim the bounty. But the issue has been created successfully. Getting there with baby steps!

PS. I can provide screenshots to illustrate the points above, if required.

**Comment by TimVanMourik** [comment]
And one more minor thing. The issue asks for a "GITHUB USERNAME:" The placeholder includes an 2-sign (@you), so I filled in '@TimVanMourik'. But now the (my issue)[https://gitcoin.co/funding/details?url=https://github.com/TimVanMourik/Porcupine/issues/18] lists my GitHub profile with a double-@.
@suggestion: automatically trim the @ if the user provides one in the profile.

**Comment by gitcoinbot** [comment]
⚡️ A tip worth 0.09  ETH has been granted to @TimVanMourik for this issue from @owocki. ⚡️ 

Nice work @TimVanMourik, check your email for further instructions. | <a href='https://gitcoin.co/tip'>Send a Tip</a>

**Comment by owocki** [comment]
gonna reject this claim and instead send you a tip.  thanks for the feedback!

**Comment by omnidan** [comment]
some things I noticed with...

the UI/X in general:

- everything in caps makes it hard to parse the interface. I know this is a style decision, but it really makes it hard to see the structure in the UI. consider only using caps for headings, not everything.
- consider using formatting (like bold) to highlight certain information

the tipping UI/X:

- why is "ETH" even a dropdown if you can only select ethereum right now? just write "amount of <field> ETH" (after installing metamask, it actually shows multiple currencies)
- consider polling the github api to autocomplete usernames/real names for the username field
- when clicking send, it shows a notification "You must install metamask to use this tool", but there is no link to metamask
- the "you must install" notifications also stack when you press "send" multiple times

funding issue UI/X:

- completely different UI form the tipping UI, could be confusing.
- here you are given the option to install metamask or submit without web3 (consider doing something similar when pressing "send" in the tipping UI)
- the "submit without web3" form is a google form, maybe the UI could be combined here -> create the funded issue via the wizard, then "submit without web3" at the end, which submits via google form or some REST service that notifies the admins
- it was not possible for me to check the rest of the UI/X, because I do not have any ether so it wouldn't let me continue (in the tipping UI, it actually would let me continue and then at the end metamask told me I don't have enough funds - this is probably the better way to go about it, or at least only show it during the "pay" stage)

**Comment by omnidan** [comment]
regarding the default expiration, maybe make it a month and always show the field (even in the tipping screen)? that way it would be clear that it expires at all, and when it expires by default (it wasn't clear until I pressed "advanced" in the tipping UI). if the user isn't okay with the default, they can always change it as long as it's visible!

**Comment by nbanmp** [comment]
The "I HAVE READ, UNDERSTAND, AND AGREE TO, THE TERMS OF SERVICE" checkbox is slightly confusing. I was wondering where to view the Terms of Service, and only after waving my mouse around for a bit did I realize that "TERMS OF SERVICE" was a hyperlink. Perhaps it should be underlined by default.

Hovering over the drop-down menu results in an unhelpful popup: https://i.imgur.com/bM5vOaf.png

I funded the issue with 0.0001 ETH, which is about equal to 0.03 cents. After deploying, the "issue details" page displayed 0 ETH and 0.03 USDT. It does show the correct value on my profile page.

When I do hover over the issue on my profile page, however, it appears to attempt to underline the tags and time since the issue was created. https://i.imgur.com/pxrkXsw.png Also, that line itself is confusing: "10 MINUTES AGO 2 OTHER BEGINNER" It would be easier to read with more punctuation and/or more descriptive wording.

There also appears to be an issue with the markdown formatting of the issue description. Here is a snippet of the markdown in my issue: 
```
Currently, the only requirement is `dill`.
```
It seems that the formatting on the "issue details" page inserted an extra space: https://i.imgur.com/oGu9C8Y.png

I do not know if there is any caching, but when I edited my issue, the description page did not update within 15 minutes.

When I was funding the issue, I did not specify a "Github profile," but under "FUNDER INFO," there was still a section "GITHUB PROFILE:" and there was nothing after the colon. The web app should already know my username (it is on my profile page), so it should either fill it out or remove the line completely. 

Now that I think about it, the app has no way of knowing that it was my GitHub account that funded the issue. I could have funded an issue that I did not create, but the creator of the issue would be given a profile page that implies that they funded the issue.

**Comment by owocki** [comment]
@omnidan @nbanmp sorry for the radio silence.  been heads down on a few other things. i  think i still owe you your payouts for this (great) feedback, right?

**Comment by gitcoinbot** [comment]
⚡️ A tip worth 0.09  ETH has been granted to @omnidan  for this issue. ⚡️ 

Nice work @omnidan , check your email for further instructions. | <a href='https://gitcoin.co/tip'>Send a Tip</a>

**Comment by owocki** [comment]
tips paid out.  enjoy, folks!

closing this feedback bounty for now.

**Comment by owocki** [comment]
@omnidan @nbanmp let me know if you dont get emails about your tips.  been having a few issues with some ppl not getting their emails due to some restrictions in the github api

**Comment by nbanmp** [comment]
@owocki I don't see any emails.

**Comment by gitcoinbot** [comment]
⚡️ A tip worth 0.09  ETH has been granted to @nbanmp for this issue. ⚡️ 

Nice work @nbanmp, check your email for further instructions. | <a href='https://gitcoin.co/tip'>Send a Tip</a>

**Comment by nbanmp** [comment]
Just got it.

**Comment by nbanmp** [comment]
I got it.
